src/Polysemy/Internal.hs

```diff
@@ -342,7 +352,7 @@ hoistSem nat (Sem m) = Sem $ \k -> m $ \u -> k $ nat u
 
 
 ------------------------------------------------------------------------------
--- | Introduce an arbitrary number of effects on top of the effect stack. This
+-- | Introduce an arbitrary number of effects on head of the effect stack. This
```

Comment:
the head

Reply:
whoops, that was a _replace all_ :sweat_smile:

Reply:
changing it back to the original wording

Reply:
maybe _stack_ is a misleading term, given that we're inserting effects in the middle of the list